be/src/runtime/data_stream_recvr.cc

```diff
@@ -506,8 +506,13 @@ Status DataStreamRecvr::SenderQueue::_deserialize_chunk(const ChunkPB& pchunk, v
     size_t serialized_size = pchunk.serialized_size();
     if (pchunk.compress_type() == CompressionTypePB::NO_COMPRESSION) {
         SCOPED_TIMER(_recvr->_deserialize_row_batch_timer);
-        TRY_CATCH_BAD_ALLOC(RETURN_IF_ERROR(chunk->deserialize((const uint8_t*)pchunk.data().data(),
-                                                               pchunk.data().size(), _chunk_meta, serialized_size)));
+        io::ArrayInputStream is(reinterpret_cast<const uint8_t*>(pchunk.data().data()),
```

Comment:
chunk data size could larger than int max.

Reply:
> chunk data size could larger than int max.

No, it's impossible, because the `pchunk` is a protobuf object, and protobuf itself is using the ZeroCopy{InputStrea, OutputStream} to serialize and deserialize messages, so it's impossible to have a `pchunk` with size greater than INT_MAX

Reply:
Reference: https://stackoverflow.com/questions/34128872/google-protobuf-maximum-size